src/wizards/clusters.ts

```diff
@@ -127,6 +127,14 @@ export async function addClusterWizard(clusterSettings: ClusterSettings, explore
         });
         explorer.refresh();
         window.showInformationMessage(`Cluster '${name}' created successfully`);
+        // Select the created cluster is done with TreeView#reveal
+        // 1. Show the treeview of the explorer (otherwise reveal will not work)
+        explorer.show();
+        // 2. the call of reveal must be done in a timeout

```

Comment:
the reveal() call must occur in a timeout,

Reply:
fixed